src/lib/core/BUILD.bazel

```diff
@@ -7,7 +7,7 @@ ng_module(
   name = "core",
   srcs = glob(["**/*.ts"], exclude=["**/*.spec.ts"]),
   module_name = "@angular/material/core",
-  assets = [":pseudo_checkbox_css"],
+  assets = [":pseudo_checkbox_css", ":option_css", ":optgroup_css"],
```

Comment:
@jelbourn these changes seem to make Bazel happy, but I'm not sure whether it's the proper way to do it.